app/src/main/java/at/roteskreuz/stopcorona/model/managers/DatabaseCleanupManager.kt

```diff
@@ -72,23 +70,55 @@ class DatabaseCleanupManagerImpl(
         launch {
             val configuration = configurationRepository.observeConfiguration().blockingFirst()
 
-            val thresholdRedMessages = ZonedDateTime.now().minusHours(configuration.redWarningQuarantine?.toLong() ?: Long.MAX_VALUE)
-            infectionMessageDao.removeReceivedInfectionMessagesOlderThan(MessageType.InfectionLevel.Red, thresholdRedMessages)
-
-            val thresholdYellowMessages = ZonedDateTime.now().minusHours(configuration.yellowWarningQuarantine?.toLong() ?: Long.MAX_VALUE)
-            infectionMessageDao.removeReceivedInfectionMessagesOlderThan(MessageType.InfectionLevel.Yellow, thresholdYellowMessages)
-
-            val thresholdGreenMessages = ZonedDateTime.now().minusDays(THRESHOLD_REMOVE_INCOMING_GREEN_MESSAGES)
-            infectionMessageDao.removeReceivedInfectionMessagesOlderThan(MessageType.Revoke.Suspicion, thresholdGreenMessages)
+            val thresholdRedMessages = ZonedDateTime.now()
+                .minusHours(configuration.redWarningQuarantine?.toLong() ?: Long.MAX_VALUE)
+            infectionMessageDao.removeReceivedInfectionMessagesOlderThan(
+                MessageType.InfectionLevel.Red,
+                thresholdRedMessages
+            )
+
+            val thresholdYellowMessages = ZonedDateTime.now()
```

Comment:
can we move all the threshholds together so they can be reviewed better

Reply:
I would say it's better to have them grouped with the method where they are used to better understand their purpose.

Reply:
Let's keep it.